File: Lesson6_Maskaikin/app/src/main/java/com/vlad/lesson6_maskaikin/Banner.java
package com.vlad.lesson6_maskaikin;

import android.widget.ImageView;
import android.widget.TextView;

public class Banner {

    private int imageBanner;
    private String stringBanner;

    public Banner(int imageBanner, String stringBanner) {
        this.imageBanner = imageBanner;
        this.stringBanner = stringBanner;
    }

    public int getImageBanner() {
        return imageBanner;
    }

    public void setImageBanner(int imageBanner) {
        this.imageBanner = imageBanner;
    }

    public String getStringBanner() {
        return stringBanner;
    }

    public void setStringBanner(String stringBanner) {
        this.stringBanner = stringBanner;
    }
}

File: Lesson6_Maskaikin_Kotlin/app/src/main/java/com/vlad/lesson6_maskaikin_kotlin/Banner.kt
package com.vlad.lesson6_maskaikin_kotlin

class Banner (private var imageBanner:Int, private var stringBanner: String)
{
    fun getImageBanner(): Int {
        return imageBanner
    }

    fun setImageBanner(imageBanner: Int) {
        this.imageBanner = imageBanner
    }

    fun getStringBanner(): String {
        return stringBanner
    }

    fun setStringBanner(stringBanner: String) {
        this.stringBanner = stringBanner
    }
}